package.json

```diff
@@ -59,21 +58,19 @@
     "nve": "18.0.0",
     "prettier": "3.4.1",
     "publint": "0.3.0",
-    "rollup": "4.31.0",
     "ts-node": "10.9.2",
     "type-coverage": "2.29.7",
     "typescript": "5.7.3",
     "yaml-eslint-parser": "1.2.2"
   },
   "scripts": {
-    "prepublishOnly": "npm run build",
```

Comment:
out of scope change but just including it here. This script is useless because [lifecycle scripts are disabled](https://github.com/ericcornelissen/eslint-plugin-top/blob/1bc2e49284683db617c8776a7aa26268c5b4db61/.npmrc#L2).